<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>731</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 68, 121);
font: 36pt &quot;Nirmala UI&quot;;</string>
    </property>
    <property name="text">
     <string>Sistema Automático de Medida</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 68, 121);
font: 18pt &quot;Nirmala UI&quot;;</string>
    </property>
    <property name="text">
     <string>Resultado</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>490</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 68, 121);
font: 15pt &quot;Nirmala UI&quot;;</string>
    </property>
    <property name="text">
     <string>Juan Ignacio Ochotorena Pérez-Argos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="otra_medida">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>470</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 15pt &quot;Nirmala UI&quot;;</string>
    </property>
    <property name="text">
     <string>Otra medida</string>
    </property>
   </widget>
   <widget class="QPushButton" name="salir">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>470</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 15pt &quot;Nirmala UI&quot;;</string>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>470</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 15pt &quot;Nirmala UI&quot;;</string>
    </property>
    <property name="text">
     <string>Cambiar Instrumentos</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>581</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>400</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 68, 121);
font: 15pt &quot;Nirmala UI&quot;;</string>
    </property>
    <property name="text">
     <string>Voltaje medido [V]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>231</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 68, 121);
font: 15pt &quot;Nirmala UI&quot;;</string>
    </property>
    <property name="text">
     <string>Voltaje  </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>231</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 68, 121);
font: 15pt &quot;Nirmala UI&quot;;</string>
    </property>
    <property name="text">
     <string>generado [V]</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777205</width>
     <height>16777215</height>
    </size>
   </property>
   <widget class="QMenu" name="menudfgagf">
    <property name="title">
     <string>Sistema Automatico de Medida</string>
    </property>
   </widget>
   <addaction name="menudfgagf"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
